darwin/cli_functions.py

```diff
@@ -366,17 +366,20 @@ def export_dataset(
     identifier: DatasetIdentifier = DatasetIdentifier.parse(dataset_slug)
     ds: RemoteDataset = client.get_remote_dataset(identifier)
 
-    ds.export(
-        annotation_class_ids=annotation_class_ids,
-        name=name,
-        include_url_token=include_url_token,
-        include_authorship=include_authorship,
-        version=version,
-    )
-
-    identifier.version = name
-    print(f"Dataset {dataset_slug} successfully exported to {identifier}")
-    print_new_version_info(client)
+    try:
```

Comment:
@Nathanjp91 mentioned this, I don't have a problem with `try:except` when used properly like this, my main aim is to remove it in SDK layers where it obscures errors that would bubble into people's homegrown code.

This being CLI wrapper, it's perfectly appropriate.